documentation/config_logging_advanced.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: "Puppet Server: Advanced Logging Configuration"
+canonical: "/puppetserver/latest/config_logging_advanced.html"
+---
+
+Puppet Server uses the very flexible, very powerful [Logback](http://logback.qos.ch/) library under the hood to handle all of our logging needs.
```

Comment:
Can we add a line to note that it's configured in `logback.xml`, located by default at `/etc/puppetserver/logback.xml` or `/etc/puppetlabs/puppetserver/logback.xml`? I know it's redundant with what's in `configuration.html`, but if people wind up on this page through a search result they won't have that context.